.eslintrc

```diff
@@ -16,6 +16,7 @@
     "import/prefer-default-export": "off",
     "max-params": "off",
     "max-classes-per-file": "off",
+    "mocha/no-exclusive-tests": "error",
```

Comment:
I can still commit files that have been modified to use `it.only`. I think our lint-staged config is broken somehow. Though it seems to notice the issue in CI, which is good enough for me.

Reply:
Fixed that as well